src/ResumeCache.cpp

```diff
@@ -46,7 +46,13 @@ ResumeCache::~ResumeCache() {
 
 void ResumeCache::trackReceivedFrame(
     const folly::IOBuf& serializedFrame,
-    const FrameType frameType) {
+    const FrameType frameType,
+    const folly::Optional<StreamId> streamIdPtr) {
+  if (frameType == FrameType::REQUEST_STREAM && streamIdPtr) {
```

Comment:
As long as this doesn't break warm resumption for channel, which is something we'll need. Note we'll need cold start resumption to also work in that it cancels the channel in that case.